CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+v1.9.0
+------
+* Remove Redis TOI dump command
+* Prune TOI adapted to postgres and standalone tileserver
```

Comment:
This was the pr title for #204. Do you think it's worth unpacking this a little bit?